src/applications/static-pages/homepage-hero-randomizer/index.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+  Hero component with veteran portraits that are in a random order on page load
+*/
+
+import React, { useEffect, useState } from 'react';
+
+// shuffle array using Fisher-Yates shuffle
+const randomizeOrder = imageArray => {
+  const tempArray = imageArray;
+  let currentIndex = tempArray.length;
+  let randomIndex;
+
+  // While there remain elements to shuffle...
+  while (currentIndex !== 0) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    // And swap it with the current element.
+    [tempArray[currentIndex], tempArray[randomIndex]] = [
+      tempArray[randomIndex],
+      tempArray[currentIndex],
+    ];
+  }
+
+  // grab first 3 items
+  return tempArray.slice(0, 3);
+};
+
+// these files live in content-build
+const imagePaths = [
+  'babu',
+  'britton',
+  'clark',
+  'cogswell',
+  'dinh',
+  'henline',
+  'pickard-locke',
+  'singleton',
+];
+
+const HeroRandom = () => {
+  // these hex values are not in the design system, using this implementation for usability testing only
+  const gradientValues = ['#105f9f', '#144e84', '#143e6a'];
+
+  const [randomized, setRandomized] = useState([]);
```

Comment:
If we do decide to go with this redesign after usability testing, then we will have to move this into a separate hook file for reusability. Won't that be tech debt? And can we guarantee a ticket will be created to move this logic into a custom hook? My concern is that this would get forgotten once the results of usability testing are finalized.

Reply:
Similar to my point above, I don't have a super strong conviction around this. Reusability is a very real concern. In this case, though, I can basically guarantee this isn't going to be reused anywhere. It's very specific. If it were to be reused, I think I'd argue that it'd need to be generalized a bit. For example, it'd likely be more useful if it were able to take the number of images as a parameter.  

I personally have a tendency to go _too much_ to that end of the spectrum. Making things reusable almost always means making things more general, and handling that complexity slows development considerably. If it's unlikely that things will be reused, I generally tend towards being specific and then generalizing when we come to that bridge.

@kind-claudio Some really good suggestions overall. If all of your changes were implemented here, I wouldn't bat an eye.